Loading local modules in TypeScript based lambda's doesn't work

Example Lambda:

```
import { APIGatewayProxyEvent, Context } from 'aws-lambda';
import { SomeModule } from './some-module';

export function handler(event: APIGatewayProxyEvent, context: Context): void {
   //...
}
```

Webpack Error:
```
   'ERROR in ./lib/app/lambda.ts\n' +
       "Module not found: Error: Can't resolve './some-module' in '/<repo-dir>/lib/app'\n" +
```

--- a/src/NodejsFunction.ts
+++ b/src/NodejsFunction.ts
@@ -112,7 +112,7 @@ export class NodejsFunction extends lambda.Function {
       target: "node",
       resolve: {
         modules: ["node_modules", "."],
-        extensions: [ '.tsx', '.ts', '.js' ],
+        extensions: [ '.ts', '.js' ],
       },
       devtool: "source-map",
       module: {

--- a/src/NodejsFunction.ts
+++ b/src/NodejsFunction.ts
@@ -111,7 +111,8 @@ export class NodejsFunction extends lambda.Function {
       entry: "${entryFullPath}",
       target: "node",
       resolve: {
-        modules: ["node_modules", "."]
+        modules: ["node_modules", "."],
+        extensions: [ '.tsx', '.ts', '.js' ],
       },
       devtool: "source-map",
       module: {

